client/pkg/pynuoadmin.py

```diff
@@ -27,26 +30,38 @@ def __init__(self):
         self.stage = self.staged[0]
 
     def prereqs(self):
-        # We need nuodb to get nuokeymanager.jar
-        return ['nuodb']
+        # We need nuodb to get nuokeymanager.jar and pynuoadmin uses pynuodb
+        return ['nuodb', 'pynuodb']
 
     def unpack(self):
+        pypi = PyPIMetadata(self.__PKGNAME)
+        self.setversion(pypi.version)
+
         rmdir(self.pkgroot)
         mkdir(self.pkgroot)
-        runpip(self.__PKGNAME, self.pkgroot)
+        pipinstall('%s[completion]==%s' % (self.__PKGNAME, pypi.version), self.pkgroot)
 
     def install(self):
-        self.stage.stage('python', ['./'])
+        # Omit the files that were installed due to pynuodb
+        self.stage.omitcontents = Package.get_package('pynuodb').staged[0].getcontents()
+
+        self.stage.stage(os.path.join('python', 'site-packages'), ['./'])
 
         nuodb = self.get_package('nuodb')
+
+        # This needs to go two levels below the root, as it contains a ../..
```

Comment:
a much simpler solution is to fix `pynuoadmin` to follow the usual pip location and install itself in a directory

Reply:
Hm.  I considered that, but is it really "simpler"?  We'd have to first make a new pynuoadmin package and post it to pypi, then update this to manage it, and all the current users of pynuoadmin would have to be changed to import the new path.

To me it seems like creating a subdirectory here is actually simpler :)

Reply:
we can do this in a separate PR, if we decide to change the structure and bump the major version

Reply:
I don't think moving the structure changes anything... it moves things down one level in the DB package too, so we'll still have to keep this extra directory level in the client package...?